app/models/maintenance_tasks/run.rb

```diff
@@ -48,7 +48,12 @@ class Run < ApplicationRecord
 
     validates_with RunStatusValidator, on: :update
 
-    has_one_attached :csv_file, service: MaintenanceTasks.active_storage_service
```

Comment:
Maybe we could use check `MaintenanceTasks.active_storage_service` instead? If it's not set, we don't pass it which works the same? I feel like it's less hack-ish than doing introspection on the method?

That doesn't solve the problem if a user tries to set the configuration and is not using Rails 6.1 but then that happens at development time, not when upgrading the gem.

Reply:
This seems sensible to me 👍 I think if the user tries to set it and it fails because they're not yet on 6.1 it's fine.